concent_api/core/validation.py

```diff
@@ -80,26 +80,28 @@ def validate_golem_message_client_authorization(golem_message: message.concents.
     validate_public_key(golem_message.client_public_key, 'client_public_key')
 
 
-def validate_list_of_identical_task_to_compute(list_of_task_to_compute: List[message.TaskToCompute]):
-    assert isinstance(list_of_task_to_compute, list)
-    assert all([isinstance(task_to_compute, message.TaskToCompute) for task_to_compute in list_of_task_to_compute])
+def all_messages_identical(golem_messages_list: List[message.Message]):
```

Comment:
I would be great if we could keep `validate_` prefix in this function name, as we have in other validation functions.

Reply:
@rwrzesien, in my opinion if it returns boolean it should be in a form of a question; my thoughts on that topic: https://github.com/golemfactory/concent/pull/322#discussion_r183947116
@kbeker: I would prefer `are_all_messages_identical`

Reply:
I'm also for dropping `validate_` prefix. That's because it's not validation in the same sense the other functions are. It's not raising validation errors but just returning the result of a comparison. It's a generic function for comparing messages. I would even of as far as to move it to utils or helpers.

Reply:
Oh, I see it's raising exceptions now. But I'm with @dybi on that. You should make up your mind on whether you want a validation (raising exceptions) or a utility function (returning booleans) and stick to it.

Reply:
So lets drop the condition which returns True and add `validation_` prefix, and assert that list has at least one element. The rest of the function should work just fine with those changes.

Reply:
Sounds fine for me.

Reply:
Done